.github/workflows/notify.yml

```
@@ -0,0 +1,10 @@
+name: Draco Insights Dashboard Notify
+on:
+  pull_request:
+  paths:
+    - 'locales/**/*'
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+    - run: .github/notify.sh
```

```suggestion
    - run: curl -X POST -H 'Content-type: application/json' --data '{"text":"Translation Push Created"}' ${SLACK_WEBHOOK}
      env:
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

(You'll need to set `SLACK_WEBHOOK` in https://github.com/Brightspace/insights-engagement-dashboard/settings/secrets/actions)

---

I think calling a script should work too. I just did inline to show how the env var is used.